### ISSUE TITLE
EscapeOutputSniff should not flag echoing WP_Widget::get_field_id() and  WP_Widget::get_field_name()

### ISSUE THREAD

**Issue by westonruter** [WordPress-Coding-Standards/WordPress-Coding-Standards#159]
There's no reason that the `echo` statements on these lines should be flagged:

``` php
<label for="<?php echo $this->get_field_id( 'title' ); ?>"><?php _e( 'Title:' ); ?></label>
<input class="widefat" id="<?php echo $this->get_field_id( 'title' ); ?>" 
       name="<?php echo $this->get_field_name( 'title' ); ?>"
       type="text"
       value="<?php echo esc_attr( $instance['title'] ); ?>">
```
